src/components/StatusBadge/StatusBadge.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from "react";
+import bem from "../../utils/bem";
+
+export interface StatusBadgeProps {
+    /** BlockName for use with BEM. See how to work with blockNames and BEM here: http://getbem.com/introduction/ */
+    blockName?: string;
+    /** ClassName that appears in addition to "status-badge" */
+    className?: string;
+    /** Inner text of the element */
+    text?: string;
+    /** Type of badge **/
+    type?: string;
+}
+
+export default function Heading(props: React.PropsWithChildren<StatusBadgeProps>) {
+    const {
+        blockName,
+        className,
+        text,
+        type,
+    } = props;
+
+    let status_badge_base_class = "heading";
```

Comment:
I'd make this `.status-badge` to match what's in the [SCSS file](https://github.com/NYPL/nypl-design-system/blob/development/src/components/StatusBadge/_StatusBadge.scss).